modules/Users/entryPointSaveGoogleApiKey.php

```diff
@@ -0,0 +1,61 @@
+<?php
```

Comment:
can you please make this entry point to more 'object oriented' using less superglobals

Reply:
@gymad I'm not sure what to do here. This code takes all it's data from the _REQUEST after Google redirects to it after authorization. Is there another, better way to get the data?